Program.cs

```diff
@@ -58,34 +70,12 @@
     return grid;
```

Comment:
Read the formatting guidelines - Never use `for` without the surrounding brackets. In this method it should be:
```
for (;;) 
{
   for(;;)
   {
        ...
   }
}
```